doc/specs/mesh.md

```diff
@@ -146,8 +164,1521 @@ This guide assume you already have a Servicer properly setup and running. If not
+
+### Metrics
+
+* `moniker` helpful label to identify individual mesh instances across metrics
+* `stat_type` could be `metric` or `node`
+  * `metric` values of the worker pool to dispatch metrics to prometheus
+  * `node` values of the worker used to notify relays to node
+* `is_notify` indicate if the metric is result of the call from chain to mesh (aka relay) OR mesh to node (aka notify) 
+* `status_type` indicate where in the code the error happen, expected values are: 
+  * `internal` unexpected error like parsing/read values like json
+  * `notify` error handled in the process of notify the node about a relay 
+  * `chain` error handled on the call to the blockchain
+* `servicer_address` label is optional, you can add or remove from metrics in case you want to reduce the cardinality of the metrics.
+To do it set `metrics_attach_servicer_label` to `false` on config.json
+
+| Name | Type | Labels | Description |
+|---|---|---|---|
+| pocketcore_geo_mesh_workers_running | gauge | moniker, stat_type, node_name | Number of running worker goroutines |
+| pocketcore_geo_mesh_workers_idle | gauge | moniker, stat_type, node_name | Number of idle worker goroutines |
+| pocketcore_geo_mesh_tasks_submitted_total | gauge | moniker, stat_type, node_name | Number of tasks submitted |
+| pocketcore_geo_mesh_tasks_waiting_total | gauge | moniker, stat_type, node_name | Number of tasks waiting in the queue |
+| pocketcore_geo_mesh_tasks_successful_total | gauge | moniker, stat_type, node_name | Number of tasks that completed successfully |
+| pocketcore_geo_mesh_tasks_failed_total | gauge | moniker, stat_type, node_name | Number of tasks that completed with panic |
+| pocketcore_geo_mesh_tasks_completed_total | gauge | moniker, stat_type, node_name | Number of tasks that completed either successfully or with panic |
+| pocketcore_geo_mesh_min_workers | gauge | moniker, stat_type, node_name | Number min workers of node pool |
+| pocketcore_geo_mesh_max_workers | gauge | moniker, stat_type, node_name | Number max workers of node pool |
+| pocketcore_geo_mesh_max_capacity | gauge | moniker, stat_type, node_name | Number max capacity of node pool |
+| pocketcore_geo_mesh_relay_count | counter | moniker, chain_id, chain_name, is_notify, servicer_address (optional) | Number of relays executed |
+| pocketcore_geo_mesh_relay_time | histogram | moniker, chain_id, chain_name, is_notify, servicer_address (optional) | Relay duration in milliseconds |
+| pocketcore_geo_mesh_error_count | counter | moniker, chain_id, chain_name, is_notify, status_type, status_code, servicer_address (optional) | Number of errors resulting from relays (mesh or chain) |
+
+#### Metrics Grafana Dashboard
+
+This dashboard allow you to monitor your mesh nodes across regions, so to properly work every metric need to contains an extra label called `region`
+You can add it on your scrape job with the following rule:
+```yml
```

Comment:
add  dropdown / spoiler / new file for this

Reply:
solved as a new file